ServerCore/Pages/Teams/Play.cshtml

```diff
@@ -100,3 +104,5 @@
     }
     </tbody>
 </table>
+
+<p>Note: Puzzles with a &#x26A0; before the name have errata, which will be provided on that puzzle's answer submission page.</p>
```

Comment:
Should this be at the top? This page can get long

Reply:
I don't think people need to look at it all the time. It'll be shorter initially and people will see what it means then.